apertium-tur-uzb.tur-uzb.dix

```diff
@@ -2468,15 +2481,21 @@
 <e><p><l>topla<s n="v"/><s n="tv"/></l><r>toʻpla<s n="v"/><s n="tv"/></r></p></e>
 <e><p><l>seç<s n="v"/><s n="tv"/></l><r>tanla<s n="v"/><s n="tv"/></r></p></e>
 <e><p><l>ölçül<s n="v"/><s n="iv"/></l><r>oʻlchan<s n="v"/><s n="iv"/></r></p></e>
-<e><p><l>örüş<s n="v"/><s n="iv"/></l><r>toʻqish<s n="v"/><s n="iv"/></r></p></e>
+<e><p><l>ör<s n="v"/><s n="iv"/></l><r>toʻqi<s n="v"/><s n="iv"/></r></p></e>
 <e><p><l>dur<s n="v"/><s n="iv"/></l><r>tur<s n="v"/><s n="iv"/></r></p></e>
-<e><p><l>doğ<s n="v"/><s n="iv"/></l><r>tugʻil<s n="v"/><s n="iv"/></r></p></e>
-<e><p><l>doğ<s n="v"/><s n="iv"/></l><r>tavallud<b/>top<s n="v"/><s n="iv"/></r></p></e>
+<e><p><l>doğ<s n="v"/><s n="tv"/></l><r>tugʻil<s n="v"/><s n="tv"/></r></p></e>
+<e><p><l>doğ<s n="v"/><s n="tv"/></l><r>tavallud<b/>top<s n="v"/><s n="tv"/></r></p></e>
```

Comment:
These should be iv, no?

Reply:
yeah it is iv, my fault I thought as we say it "uşağı doğdu", in Turkish "çocuğu doğurdu"

Reply:
Changing them back to "iv" then.